deploy/docker/.env

```diff
@@ -17,6 +17,7 @@
 #
 HUB=ghcr.io/apache/dolphinscheduler
 TAG=latest
+DOCKER=TRUE
```

Comment:
This is already set in all `Dockerfile`

https://github.com/apache/dolphinscheduler/blob/67cc260d52c1f2e5aa7db76aa8621cdd0f8c4ee0/dolphinscheduler-tools/src/main/docker/Dockerfile#L20-L21

Reply:
> This is already set in all `Dockerfile`
> 
> https://github.com/apache/dolphinscheduler/blob/67cc260d52c1f2e5aa7db76aa8621cdd0f8c4ee0/dolphinscheduler-tools/src/main/docker/Dockerfile#L20-L21

I had updated and thanks for your reminder, YYDS.